### ISSUE TITLE
Formatted raw strings no longer work in 2021.5.0

### ISSUE THREAD

**Issue by nathanrpage97** [microsoft/pylance-release#1241]
## Environment data

-   Language Server version: XXX
-   OS and version: XXX

## Expected behaviour

formatted raw strings were previously supported, but are now broken.

## Actual behaviour

XXX

## Code Snippet / Additional information

```python
data = "test"
x = fr"{data}\s" # breaks in 2021.5.0 (says data is unused, and improperly renders)
```

**Comment by jakebailey** [comment]
This is the same as https://github.com/microsoft/pyright/issues/1830 and will be fixed in the next release.

**Comment by nathanrpage97** [comment]
Thanks!

**Comment by jakebailey** [comment]
Leaving open until we release the fix (shortly).

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.5.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202151-6-may-2021
